src/Components/SectionHeader.elm

```diff
@@ -116,44 +126,46 @@ viewArtists buttonMsg artist =
         ]
 
 
-viewVideos : List VideoObj -> Element msg
-viewVideos videoslist =
-    wrappedRow [ Element.height fill, Element.width fill, paddingXY 5 5, spacingXY 5 7 ]
-        (List.map
-            (\video ->
-                column
-                    [ paddingXY 5 5
-                    , Background.color (rgb 1 1 1)
-                    , mouseOver [ Background.color Colors.sidebar ]
-                    , Element.height (fill |> minimum 170 |> maximum 170)
-                    , Element.width (fill |> minimum 280 |> maximum 280)
-                    , Border.rounded 3
-                    , clipX
-
-                    --, Element.below (videoLabel video)
-                    ]
-                    [ case video.thumbnail of
-                        "" ->
-                            image [ width fill, height fill, Element.inFront (videoLabel video) ]
-                                { src = "https://via.placeholder.com/170"
-                                , description = "Hero Image"
-                                }
-
-                        _ ->
-                            image [ width fill, height fill, Element.inFront (videoLabel video) ]
-                                { src = crossOrigin "http://localhost:8080" [ "image", percentEncode video.thumbnail ] []
-                                , description = "Thumbnail"
-                                }
-                    , videoLabel video
-                    ]
-            )
-            videoslist
-        )
+viewVideos : msg -> VideoObj -> Element msg
+viewVideos buttonMsg video =
+    column
+        [ Background.color (rgb 1 1 1)
+        , Element.height (fill |> minimum 130 |> maximum 180)
+        , Element.width (fill |> minimum 280 |> maximum 280)
+        , Element.htmlAttribute (Html.Attributes.class "card-parent")
+        ]
+        [ case video.thumbnail of
+            "" ->
+                image [ width fill, height (px 125) ]
+                    { src = "https://via.placeholder.com/170"
```

Comment:
We need to come up with a strategy for this. Probably better sooner than later. As we can't rely on external pages like that.